### PR TITLE
DOMA-2914 fixed getOne issue on muptiple residents for same user-organization-property

### PR DESCRIPTION
**Pull request by akarjakin**

### CODE REVIEW

**Review by sitozzz — APPROVED** [review]

**Review by nomerdvadcatpyat — APPROVED** [review]

File: apps/condo/domains/ticket/utils/handlers.js [hunk 1]
```diff
@@ -192,11 +192,12 @@ const sendTicketNotifications = async (requestData) => {
         }
 
         if (ticketStatusType) {
-            const { property: propertyId, organization: organizationId } = updatedItem
+            const { property: propertyId, organization: organizationId, unitName } = updatedItem
             const where = {
                 user: { id: clientId },
                 property: { id: propertyId },
                 organization: { id: organizationId },
+                unitName,
```
Comment (pahaz) [inline apps/condo/domains/ticket/utils/handlers.js:200]: What abou unitType?

Comment (akarjakin) [reply apps/condo/domains/ticket/utils/handlers.js:200]: added

**Comment by pahaz** [comment]
You need to add a test where multiple residents live in one unitName :)

File: apps/condo/domains/ticket/utils/handlers.js [hunk 2]
```diff
@@ -247,6 +251,8 @@ const sendTicketCommentNotifications = async (requestData) => {
             user: { id: clientId },
             property: { id: propertyId },
             organization: { id: organizationId },
+            unitName,
```
Comment (pahaz) [inline apps/condo/domains/ticket/utils/handlers.js:254]: You need to check that we have a constrain for that

**Comment by akarjakin** [comment]
> 

different users with same unit address would have different `clientId`

File: apps/condo/domains/ticket/utils/handlers.js (hunk 2 above)
Comment (pahaz) [reply apps/condo/domains/ticket/utils/handlers.js:254]: Please add the constrain in other PR

Comment (akarjakin) [reply apps/condo/domains/ticket/utils/handlers.js:254]: Added ticket for that DOMA-2916

**Review by AntonAL — APPROVED** [review]

**Pull request closed without merging**